client/deposits/index.tsx

```diff
@@ -38,10 +41,24 @@ const useNextDepositNoticeState = () => {
 	};
 };
 
-const NextDepositNotice: React.FC = () => {
+const useAccountStatus = () => {
```

Comment:
Changes in this PR can use a behavior test, wdyt?

Reply:
The Deposit Listing screen has no test file, will create a issue to add tests for this file, including this functionality

Reply:
Created issue here https://github.com/Automattic/woocommerce-payments/issues/8417